src/melee/ft/ftcoll.c

```diff
@@ -68,6 +69,9 @@ struct UnkSize320_t {
 static DmgLogEntry dmg_log0[20];
 struct DmgLogEntry dmg_log1[20];
 
+// .sdata
+char* const ftColl_804D3A68;
+
```

Comment:
Unused? I think this came from a merge with another PR and is not needed anymore